arcgis-ios-sdk-samples/Content Display Logic/Views/AppInfoView.swift

```diff
@@ -0,0 +1,170 @@
+// Copyright 2021 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct AppInfoView: View {
+    private let links = [
+        Link(title: "Esri Community", url: URLs.esriCommunity, subtitle: "Browse and discuss in the Esri Community"),
+        Link(title: "GitHub Repository", url: URLs.githubRepository, subtitle: "Log an issue in the GitHub Repository"),
+        Link(title: "API Reference", url: URLs.apiReference)
+    ]
+    
+    private let appInfos = [
+        AppInfo(title: "App Version", detail: Strings.appMarketingVersionNumber),
+        AppInfo(title: "SDK Version", detail: Strings.ArcGISSDKVersionString)
+    ]
+    
+    private let poweredBys = [
+        Link(title: "ArcGIS Runtime Toolkit for iOS", url: URLs.toolkit),
+        Link(title: "ArcGIS Runtime SDK for iOS", url: URLs.developers)
+    ]
+    
+    private let copyrightFooter: String = {
+        let currentYear = Calendar.current.component(.year, from: Date())
+        return String(format: "Copyright © 2015-%d Esri. All Rights Reserved.", currentYear)
+    }()
+    
+    private func header(for link: Link) -> String? {
+        if link == links.first {
+            return "Useful Links"
+        } else {
+            return nil
+        }
+    }
+    
+    var body: some View {
+        List {
+            HStack {
+                Spacer()
+                VStack {
+                    Image("AppIcon2", bundle: .main)
+                    Text(Strings.appName)
+                        .font(.headline)
+                }
+                Spacer()
+            }
```

Comment:
Instead of having the `copyrightText` as the section footer, shall we put it next to the bundle name so that it is center aligned? Because the section footer aligns to the left by default.

![image](https://user-images.githubusercontent.com/9660181/148107093-f4f964f4-790c-450b-bf9e-da334dd10419.png)

> Provide a label for the image (used for accessibility).
Great practice learned. 👍